jekyll/_cci2/pipelines.adoc

```diff
@@ -53,6 +53,7 @@ image::project-settings-pipelines.png[Pipelines page in project settings]
 TIP: If you created a project xref:create-project#[in the CircleCI web app] you will find a pipeline is already set up.
 
 image::add-pipeline.png[Add a pipeline]
+#rosie to update image#
```

Comment:
The new form to create a Github App pipeline does not include the field "fallback branch". 
Example:
<img width="1050" alt="Screenshot 2024-10-14 at 19 01 21" src="https://github.com/user-attachments/assets/374bf953-89a3-4e00-b1b0-cf66adf57141">
this is live!